packages/react-conformance/src/types.ts

```diff
@@ -5,10 +5,16 @@ import { mount } from 'enzyme';
 
 export type Tests = keyof typeof defaultTests;
 
+export interface TestingOptions {
+  'consistent-callback-names'?: {
+    ignoreProps?: string[];
+  };
+}
+
 /**
  * {@docCategory isConformant}
```

Comment:
These interfaces don't need the docCategory tag (that's only for things that will be on the public doc site)